test/integration_test.py

```diff
@@ -1008,22 +1008,23 @@ def assert_manifest(expected_bundles):
 
             # Create a single-file curl manifest and verify that the OAuth2
             # token is present in the .curlrc
-            managed_access_file = self.random.choice(self.random.choice(hits)['files'])
-            manifest_url.set(args={
-                'filters': json.dumps({'fileId': {'is': [managed_access_file['uuid']]}}),
-                'format': 'curl'
-            })
-            with self._service_account_credentials:
-                response = self._get_url_content(str(manifest_url))
-            self._check_curl_manifest(catalog, response)
-            auth_header_line = one(
-                line
-                for line in TextIOWrapper(BytesIO(response))
-                if line.startswith('--header')
-            )
-            auth_header = auth_header_line.split(' ', 1).pop().strip('"\n')
-            self.assertEqual(f'Authorization: Bearer {self._tdr_client.credentials.token}',
-                             auth_header)
+            if hits:
```

Comment:
```suggestion
            if hits:
```

A subsequent, unrelated issue might force us to backport this, so it should be conditional on the `prod` and `prod2` deployments, as an additional safe-guard against reducing coverage on `dev` and `sandbox`, should this change ever show up there. All under the assumption that this is only the temporary fix, of course.